pkg/platform/api/buildplanner/request/mergecommit.go

```diff
@@ -0,0 +1,76 @@
+package request
+
+import "github.com/ActiveState/cli/pkg/platform/api/buildplanner/model"
+
+func MergeCommit(owner, project, targetRef, otherRef string, strategy model.MergeStrategy) *mergeCommit {
+	return &mergeCommit{map[string]interface{}{
+		"organization": owner,
+		"project":      project,
+		"targetRef":    targetRef,
+		"otherRef":     otherRef,
+		"strategy":     strategy,
+	}}
+}
+
+type mergeCommit struct {
+	vars map[string]interface{}
+}
+
+func (b *mergeCommit) Query() string {
+	return `
+mutation ($organization: String!, $project: String!, $targetRef: String!, $otherRef: String!, $strategy: MergeStrategy) {
+  mergeCommit(input:{organization:$organization, project:$project, targetVcsRef:$targetRef, otherVcsRef:$otherRef, strategy:$strategy}) {
+		... on MergedCommit {
+			commit {
+				__typename
+				commitId
+			}
+		}
+    ... on MergeConflict {
+      __typename
+      message
+    }
+    ... on FastForwardError {
+      __typename
+      message
```

Comment:
nit: we could query all of the fields for the error types so when we implement the user-facing errors we don't have to update the query. I'll leave it up to you though.